android/app/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,6 @@
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.RECORD_AUDIO"/>
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Do we also want to delete READ external storage or is that used by one of our packages?

Reply:
Good call